src/common/types.h

```diff
@@ -131,7 +131,7 @@ do { \
     default: ABORT("Unknown type {}", type); \
   } \
 } while(0)
-
+/// namespace marian
```

Comment:
I would not add such obvious comments just to make the doxygen happy. Why is it needed?

Reply:
This comment tells doxygen to get this namespace documented. Otherwise, Type and DeviceType cannot be documented.

Reply:
OK, thanks.